docs/docsource/operation/XDM_GUI/location_management/adding_a_storage_location.rst

```diff
@@ -0,0 +1,278 @@
+
+Target Helper Bucket for Multi-Part Uploads
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Google Cloud Storage solution imposes limitations on uploads that require
+specific workarounds. Among these is a 5 GB hard limit on uploads per command,
+which requires objects over this limit to be broken up, uploaded in parallel
+chunks, and on a successful upload reassembled in the cloud. |product| manages this
+complexity, in part, by using a "helper" bucket.
+
+.. note::
+
+   Google Cloud Storage also imposes a 1024-part cap on objects stored to its
+   locations (For all other backends, |product| caps the number of parts at
+   10,000). For data stored directly to GCP as the primary cloud, |product|
+   propagates this limitation forward to any other cloud storage services to
+   which Google data is replicated.
+
+Other Services: |product| Local, RING/sproxyd, and NFS
+------------------------------------------------------
+
+|product| Local Filesystem
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|product| Local Filesystem has similar authentication requirements to AWS S3, but
+because it is a |product|-native filesystem, it shares authentication and related
+credentialing tasks, which are addressed elsewhere in the |product| UI.
+
+For more information, see :ref:`Zenko Local`.
+
+RING with sproxyd Connector
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The RING maintains stability and redundancy in its object data stores by way of
+a bootstrap list. To access a RING directly using sproxyd, you must enter at
+least one bootstrap server; however, more is better.  This is simply a list of
+IP addresses for the bootstrap servers in the RING. The order of entry is not
+important: none enjoys a preferred position. Entries must assign a port
+number. If a port number is not explicitly assigned, |product| assigns port 8081 by
+default. Entries can use DNS or IP address format.
+
+NFS
+~~~
+
+|product| supports out-of-band updates from NFSv3 and NFSv4 file systems. |product|
+replicates data from NFS servers to cloud storage services using scheduled cron
+jobs.
+
+.. note::
+
+   For NFS mounts, |product| cannot perform data PUT transactions. In other words,
+   data can be written directly to NFS for |product| to replicate to other backends,
+   but cannot be written to |product| to replicate to NFS.
+
+Configuring NFS requires you to specify the transfer protocol (TCP or UDP), NFS
+version (v3 or v4), the server location (IP address or URI), export path (the
+path to the NFS mount point on the server), and the desired NFS options (rw and
+async are the default entries).
+
+AWS
+~~~
+
+|product| can ingest metadata out of band from AWS in much the same way it can
+ingest out-of-band updates from NFS mounts. AWS metadata is ingested in an
+initial setup, then changes are mapped via a regularly scheduled cron
+job. |product| develops its own namespace for the Amazon bucket and can perform
+metadata-\ related tasks (CRR, metadata search, lifecycle management, etc.) on
```

Comment:
any reason why this dash word is doing `one\- two` compared to all the other ones just being `one-two`?

Reply:
That was a line break/continuation that got shifted off the line end. 

Good spot. Fixed.